### PR TITLE
Fix: Add missing karma service and configuration

### PR DESCRIPTION
**Pull request by oxedions**
Karma was partially missing from the prometheus_server role.

karma.yml file is from @sobrase

### CODE REVIEW

**Review by osmocl — APPROVED** [review]
Ok for me.

File: roles/addons/prometheus_server/templates/karma.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,73 @@
+authentication:
+  basicAuth:
+    users:
+      - username: admin
+        password: admin
```
Comment (btravouillon) [inline roles/addons/prometheus_server/templates/karma.yml.j2:5]: Can you create tunable here?

File: roles/addons/prometheus_server/templates/karma.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+alertmanager:
+  interval: 10s
+  servers:
+    - cluster: prometheus
+      name: prometheus1
```
Comment (btravouillon) [inline roles/addons/prometheus_server/templates/karma.yml.j2:11]: tunable?

File: roles/addons/prometheus_server/templates/karma.yml.j2 [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      timeout: 10s
+      proxy: false
+
+#filters by default when connected      
+filters:
+  default:
+    - "@cluster=prometheus"
+
+karma:
+  name: alertmanager_karma
```
Comment (btravouillon) [inline roles/addons/prometheus_server/templates/karma.yml.j2:22]: tunable?

**Comment by oxedions** [comment]
Important: the current "defaults" variables are for a single alertmanager to monitor.

In the futur, if we desire to be able to gather more alertmanagers from karma, a small update of the role will be needed.

**Pull request merged**